synapse/handlers/identity.py

```diff
@@ -250,19 +253,20 @@ async def try_unbind_threepid(self, mxid: str, threepid: dict) -> bool:
         changed = True
         for id_server in id_servers:
             changed &= await self.try_unbind_threepid_with_id_server(
-                mxid, threepid, id_server
+                mxid, medium, address, id_server
             )
 
         return changed
 
     async def try_unbind_threepid_with_id_server(
```

Comment:
Should this be private? (prefix with a `_`)?

Reply:
Yes!